style.css

```diff
@@ -0,0 +1,267 @@
+.wrapper {
+  width: 100vm;
+  padding-top: 30px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: baseline;
+}
+
+.menu-button {
+  padding-right: 20px;
+}
+
+.headline {
+  background-image: url('header-shapes-mobile.svg');
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  padding-left: 10px;
+  padding-top: 10px;
+  padding-bottom: 25px;
+  border-bottom-left-radius: 50px;
+  shape-margin: 1px;
+}
+
+.logo {
+  font-family: Poppins, Verdana, Geneva, Tahoma, sans-serif;
+  font-weight: 700;
+  font-size: 18px;
+  color: #6070ff;
+  padding-top: 10px;
+  padding-left: 12px;
+}
+
+.title1 {
+  font-family: Poppins, Geneva, Tahoma, sans-serif;
+  font-weight: 700;
+  font-size: 30px;
+  line-height: 52px;
+  padding-top: 50px;
+  color: #172b4d;
+}
+
+.description {
+  padding-right: 10px;
+  font-family: Poppins, Geneva, Tahoma, sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  color: #344563;
+}
+
+.connect {
+  font-family: Poppins, Verdana, Geneva, Tahoma, sans-serif;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+a:hover { color: #7f8cff; }
+
+.social {
+  display: flex;
+  width: 300px;
+  justify-content: flex-start;
+}
+
+.social ul {
+  list-style: none;
+  padding-left: 0%;
+}
+
+.social li {
+  float: left;
+  margin-inline-start: 15px;
+}
+
+.works-container {
+  display: grid;
+  width: 100%;
+  margin: 0 auto;
+  padding-top: 50px;
+  grid-template-columns: auto;
+}
+
+.snapshot {
+  margin-left: auto;
+  margin-right: auto;
+  display: block;
+  width: 95%;
+  height: auto;
+}
+
+.flex-card {
+  width: 100%;
+  flex-direction: row;
+  border-radius: 25px;
+  padding-bottom: 20px;
+  padding-top: 50px;
+}
```

Comment:
- [Optional] Please kindly consider adding a border to your work cards to match the design template. Also, add a background color to the work section to match the template,
## Your design
![m1](https://user-images.githubusercontent.com/94551484/186768904-6cf61224-5862-4f7a-acac-83a9064c5386.png)
## Figma template
![ccc](https://user-images.githubusercontent.com/94551484/186768946-4c5d6da8-6c30-4539-aafd-847acb283aac.png)